cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go

```diff
@@ -35,6 +35,9 @@ limitations under the License.
 //   - Add `ClusterConfiguration.EncryptionAlgorithm` that can be used to set the asymmetric encryption algorithm
 //     used for this cluster's keys and certificates. Can be "RSA" (default algorithm, key size is 2048) or
 //     "ECDSA" (uses the P-256 elliptic curve).
+//   - Add `ClusterConfiguration.DNS.Disabled` and `ClusterConfiguration.Proxy.Disabled` that can be used to disable
+//     the CoreDNS and kube-proxy addons during cluster initialization. Skipping the related addons phases,
+//     during cluster creation will set the same fields to `false`.
```

Comment:
note to myself: we need to update the website once v1beta4 is open.

Reply:
yep, there is authored content we need to update. the api godocs are auto-generated though.